tracer/src/Datadog.Trace/ClrProfiler/NativeMethods.cs

```diff
@@ -183,7 +129,7 @@ public static int RegisterIastAspects(string[] aspects)
             }
         }
 
-        public static int RegisterCallTargetDefinitions(string id, NativeCallTargetDefinition2[] items, uint enabledCategories)
+        public static int RegisterCallTargetDefinitions(string id, NativeCallTargetDefinition3[] items, uint enabledCategories)
```

Comment:
I'm confused, we don't need this method do we, we shouldn't be calling it from _anywhere_ as I understand it 🤔

Reply:
This method is used to test the category enablement / disablement feature. It's just an entry point that executes the same code that is invoked with the built in definitions